### ISSUE TITLE
Missing rows in displayed image after loading bitstream

### ISSUE THREAD

**Issue by jeffmilstea** [derekmulcahy/image_nx4#1]
FYI, using iMPACT to program a bitstream can require several tries before it will display the entire image without any missing rows.

**Comment by derekmulcahy** [comment]
Yes, I agree! Currently row transfer is not synchronized with the display. 6 rows are real and 2 rows are phantoms caused by the CPLD on the panel wrapping the row counter at 8 and not 6. Until we find out how to update at the right time or reset the CPLD counter to zero the sync will be random. Any ideas for syncing?
